### PR TITLE
Create Npc Update

### PR DESCRIPTION
**Pull request by EnerGonix**
Npc now pickup player from the ground

### CODE REVIEW

**Comment by JusCampin** [comment]
No files/code included in PR.

File: Npc Update [hunk 1]
```diff
@@ -0,0 +1 @@
+Npc now pickup player from the ground
```
Comment (AndrewR3K) [inline Npc Update:1]: This file is not needed.

**Review by AndrewR3K — CHANGES_REQUESTED** [review]
You did not upload anything other than a file named "Npc Update" please remove this file and upload your changes if you wish to have them updated into our script. 

Thank you for your efforts and time!!

File: Npc Update (hunk 1 above)
Comment (EnerGonix) [reply Npc Update:1]: im new to github sry

**Pull request closed without merging**